### PR TITLE
backend: fix a bug about wrong column info

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
fix a bug about wrong column info

### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Review by glorv — COMMENTED** [review]
LGTM

File: lightning/backend/tidb.go [hunk 1]
```diff
@@ -366,7 +366,7 @@ func (be *tidbBackend) FetchRemoteTableModels(schemaName string) (tables []*mode
 			SELECT table_name, column_name
 			FROM information_schema.columns
 			WHERE table_schema = ?
-			ORDER BY table_name;
+			ORDER BY table_name, ordinal_position;
```
Comment (overvenus) [inline lightning/backend/tidb.go:369]: Could you add a test?

**Comment by kennytm** [comment]
LGTM after fixing overvenus's comment.

File: lightning/backend/tidb.go (hunk 1 above)
Comment (leoppro) [reply lightning/backend/tidb.go:369]: It's hard to add a test for it. The current test framework is based on `sqlmock`. The `sqlmock` can't support to test this case. There is one way to implement this test: import the whole tide

**Comment by kennytm** [comment]
https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/lightning_ghpr_test/detail/lightning_ghpr_test/2287/pipeline

```
[2020-10-22T08:57:36.890Z] Import using importer finished
[2020-10-22T08:57:37.149Z] TEST FAILED: OUTPUT DOES NOT CONTAIN 'set: x00,x06,x07,x09,x17,x20,x23,x24,x27,x37,x44,x46,x49,x54,x55,x58,x61,x62'
[2020-10-22T08:57:37.149Z] ____________________________________
[2020-10-22T08:57:37.149Z] [Thu Oct 22 16:57:36 CST 2020] Executing SQL: SELECT `set` FROM vt.`enum-set` WHERE `enum` = "gcc"
[2020-10-22T08:57:37.149Z] *************************** 1. row ***************************
[2020-10-22T08:57:37.149Z] set: x10,x17,x20,x23,x24,x27,x37,x44,x46,x49,x54,x55,x58,x61,x62
[2020-10-22T08:57:37.149Z] ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

This looks broken.

**Review by glorv — APPROVED** [review]
LGTM

**Pull request merged**